### ISSUE TITLE
Add port and 'secure' to SMTP config

### ISSUE THREAD

**Issue by dmitrizagidulin** [xrchat/xrchat-server#33]
Currently our SMTP config looks like this:

https://github.com/xrchat/xrchat-server/blob/master/src/services/email/email.service.ts

```
{
    host: process.env.SMTP_HOST,
    secure: true,
    auth: {
      user: process.env.SMTP_USER,
      pass: process.env.SMTP_PASS
    }
  }
```

Need to:

**1)** Add a PORT config (default to `2525`).

**2)** Load the `secure` argument from config (right now it's hardcoded to `true`. Which is the correct default for deployment, but makes it harder to do Mailtrap.io local testing).

**Comment by dmitrizagidulin** [comment]
Assigning to @Vickyogesh.

**Comment by dmitrizagidulin** [comment]
Implemented by @Vickyogesh in PR #47, closing.
